bin/ch-fromhost

```diff
@@ -323,12 +323,13 @@ if [ $cray_openmpi ]; then
                    | grep -E "/usr|/opt" \
                    | sed -E 's/^.+ => (.+) \(0x.+\)$/\1/')"
 
-    # Remove libmpi.so* from image. This works around ParaView
+    # Remove core MPI libraries from image. This works around ParaView
     # dlopen(3)-related errors that we don't understand.
-    image_libmpis=$(  "${ch_bin}/ch-run" "$image" -- /sbin/ldconfig -p \
-                    | sed -nr 's|^.* => (/.*/libmpi\.so([0-9.]+)?)$|\1|p')
-    [ "$image_libmpis" ] || fatal "can't find libmpi.so* in image"
-    for f in $image_libmpis; do
+    image_mpilibs=$(  "${ch_bin}/ch-run" "$image" -- /sbin/ldconfig -p \
+                    | sed -nr 's|^.* => (/.*/(libmpi\|libopen-rte\|libopen-pal)\.so([0-9.]+)?)$|\1|p')
```

Comment:
Man page suggests `-E` rather than `-r` "for portability". Also, if you change the regex delimiters, you can drop the backslashes before the pipes I think. E.g.:

```suggestion
                    | sed -nE 's@^.* => (/.*/(libmpi|libopen-rte|libopen-pal)\.so([0-9.]+)?)$@\1@p')
```